mypyc/test-data/run-bytes.test

```diff
@@ -98,6 +98,36 @@ def test_len() -> None:
     assert len(b) == 3
     assert len(bytes()) == 0
 
+[case testBytesSlicing]
+def test_bytes_slicing() -> None:
+    b = b'abcdefg'
+    zero = int()
+    ten = 10 + zero
+    two = 2 + zero
+    five = 5 + zero
+    seven = 7 + zero
+    assert b[:ten] == b'abcdefg'
+    assert b[0:seven] == b'abcdefg'
+    assert b[two:five] == b'cde'
+    assert b[two:two] == b''
+    assert b[-two:-two] == b''
+    assert b[-ten:(-ten+1)] == b''
+    assert b[:-two] == b'abcde'
+    assert b[:two] == b'ab'
+    assert b[:] == b'abcdefg'
+    assert b[-two:] == b'fg'
+    assert b[zero:] == b'abcdefg'
+    assert b[:zero] == b''
+    assert b[-ten:] == b'abcdefg'
+    assert b[-ten:ten] == b'abcdefg'
+    big_int: int = 1000 * 1000 * 1000 * 1000 * 1000 * 1000 * 1000
```

Comment:
Test both a large short int (e.g. `2**24`) and a long integer representation (`2**63` or larger). Test both negative and positive large integers (e.g. write a loop `for bigint in ...:` followed by the current asserts).